rust/definitions/src/helpers.rs

```diff
@@ -136,6 +165,27 @@ pub fn print_multisigner_as_base58(
     }
 }
 
+/// Turn a `ecdsa::Public` addr into an Ethereum address.
+pub fn ecdsa_public_to_eth_address(public: &ecdsa::Public) -> Result<H160> {
+    let decompressed = libsecp256k1::PublicKey::parse_slice(
+        &public.0,
+        Some(libsecp256k1::PublicKeyFormat::Compressed),
+    )?
+    .serialize();
+    let mut m = [0u8; 64];
+    m.copy_from_slice(&decompressed[1..65]);
+    Ok(H160::from(H256::from_slice(
+        KeccakHasher::hash(&m).as_bytes(),
+    )))
+}
+
+/// Print a `ecdsa::Public` into `String`.
+pub fn print_ethereum_address(public: &ecdsa::Public) -> Result<String> {
+    let account = ecdsa_public_to_eth_address(public)?;
+
+    Ok(format!("{:?}", HexDisplay::from(&account.as_bytes())))
```

Comment:
```suggestion
    Ok(format!("0x{:?}", HexDisplay::from(&account.as_bytes())))
```
This way it imports to polkadotjs apps

Reply:
tested with this

```
ethereum:0xcdac872120b6cb33bcf4b7a0a16f182d321e52fd:0x91bc6e169807aaa54802737e1c504b2577d4fafedd5a02c10293b1cd60e39527
```

still does not read QR

Reply:
Works for me
<img width="1230" alt="image" src="https://user-images.githubusercontent.com/7165377/189296875-2ee011b3-8cf9-4281-b918-0f7b2a75b927.png">

https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fmoonbeam.public.blastapi.io#/accounts

<img width="164" alt="image" src="https://user-images.githubusercontent.com/7165377/189296939-d120a6ac-067f-46bb-a3ec-50faaf4e4633.png">

Reply:
yeah it works on moonbeam, the ui doesn't accept it on moonbase alpha